gulpfile.js/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ *  The gulp tasks are split into several files in the gulp directory
+ */
+'use strict';
+var gulp = require('gulp');
+var clean = require('./gulp/clean').default;
+let images = require('./gulp/images').default;
+let scripts = require('./gulp/scripts').default;
+let styles = require('./gulp/styles').default;
+let node_modules = require('./gulp/node_modules').default;
+let runStatic = require('./gulp/static').default;
+let connect = require('./gulp/connect').default;
+let watch = require('./gulp/watch').default;
+let test = require('./gulp/test').default;
+
+exports.default = runSeries();
+exports.build = runSeries();
```

Comment:
I gave this a try and it is almost perfect. Would you be able to change it to this, please?

``` js
exports.default = gulp.series(clean, images, scripts, styles, node_modules, runStatic, connect, watch);
exports.build = gulp.series(clean, images, scripts, styles, node_modules, runStatic);
```

This way, the "build" task will end after creating the static folder in dist. The default task starts a server and watches for changes which would mean the build pipeline would not finish/have an error :)

Reply:
I see! 
Code has been updated.